### DISCUSSION TITLE
Distribution at Cost from PCBWay

### DISCUSSION THREAD

**Discussion by cannonballtaffyojones** [frntc/SIDKick-pico#83] [General]
Hi,

I have a question on distribution. I have a batch of 10 on order preassembled from PCBWay for testing, why have 1 when you can have 10 at a little extra, plus I have a fair few C64 :).

I am frustrated with the costs that people charge i for others work in the retro domain for parts and modern replacements (usually others work!).

If they work as anticipated I would like to offer them in the new service I am setting up at 1 euro over cost for the board, I have paid 100 Euros for 10. Or at cost as part of a repair.

I have maintained the not for profit or commmerical use on the PCB text. There will be no mass purchases allowed and any discovered to be inflating prices and reselling will be added to the Web Sites wall of shame. The idea of the service is to make retro computiong access easier for novices and more affordable if possible.

What are the thoughts on the acceptability of this? 

BR Gareth

**Comment by frntc** [comment]
It sounds as if you're not planning to sell them for profit, so that's fine!
